app/store/utils.js

```diff
@@ -0,0 +1,12 @@
+import Immutable from 'immutable';
+
+export const callWithStore = (callback) => {
```

Comment:
Makes sense to separate localStorage logic :+1: 

However, I still don't understand how can I get all redux setup in one place.
I need to bind keys logic in “background.js” (before “app.js” == before opening extension window), so `store` should be also created there.
Then, inside “app.js” I need to pass `store` to `<Root store={store} />`.
I didn't find a way how to use previously created store there (I even tried to save it to background window, but then it throws some strange errors) except re-creating it again, calling `createStore` function second time.
Any ideas?